helmfile.d/values/admin-namespaces-wc.yaml.gotmpl

```diff
@@ -70,6 +70,13 @@ namespaces:
       pod-security.kubernetes.io/enforce: restricted
       pod-security.kubernetes.io/warn: restricted
   {{ end }}
+  {{ if .Values.gpu.enabled }}
+  - name: gpu-operator
```

Comment:
Question: Don't we need to add this namespace to more places? Like for some grafana dashboards or something.

Reply:
I don't know, to be honest (haven't added a namespace before). I tried searching for other namespace names in the repo (like `hnc-system` and `fluentd-system`) and I don't see anything related to grafana dashboards? I did find that it should be added to `clean-wc.sh`, so I added it there.